check_diff/tests/diffy.rs

```diff
@@ -0,0 +1,22 @@
+use diffy::{self, create_patch};
+
+#[test]
+fn diffy_test_diff() {
+    let original = "The quick brown fox jumps over the lazy dog";
+    let modified = "The quick brown fox jumps over the LAZY dog";
+
+    let patch = create_patch(original, modified);
+    // diffy uses hunks which indicates the lines that are different
+    assert_eq!(patch.hunks().is_empty(), false);
+    // hence regardless, patch.to_string() will never be empty
+    assert_eq!(patch.to_string().is_empty(), false);
+}
+
+#[test]
+fn diffy_test_no_diff() {
+    let original = "The quick brown fox jumps over the lazy dog";
+
+    let patch = create_patch(original, original);
+    assert_eq!(patch.hunks().is_empty(), true);
+    assert_eq!(patch.to_string().is_empty(), false);
+}
```

Comment:
This is a good start, and definitely helps demonstrate what we can expect to get back from `create_path` when the input is the same and when it's different, but I don't think we should be directly testing a 3rd party API in this way. We should be calling our own functions e.g `CheckDiffRunners::create_diff`, and making assertions based on what they return.